angelsindustries/data.lua

```diff
@@ -35,6 +35,9 @@ require("prototypes.overrides.industries-override-functions")
 --CATEGORIES
 require("prototypes.angels-industries-category")
 
+-- ROBOTICS
+require("prototypes.angels-robotics-ghosting")
+
```

Comment:
Shouldn't this be part of angels logistics?

https://github.com/Arch666Angel/mods/blob/af09cd2434fc0f1ba3b256894189103dc66f2a89/angelsindustries/data.lua#L86-L108